server/db.js

```diff
@@ -0,0 +1,33 @@
+const mongoose = require('mongoose');
+
+mongoose.connect('mongodb://localhost/profiledb', { useNewUrlParser: true })
+    .then(() => console.log('Connected to MongoDb...'))
+    .catch(err => console.error('Could not connect to MongoDB', err))
+
+const userSchema = new mongoose.Schema({
```

Comment:
You need to set the attributes for these fields. For example, the email should be unique, both fields should be required